packages/web-pkg/src/composables/folderLink/useFolderLink.ts

```diff
@@ -67,7 +67,10 @@ export const useFolderLink = (options: ResourceRouteResolverOptions = {}) => {
       space,
       path: dirname(resource.path)
     })
-    if (isShareRoot(resource) || !parentFolderAccessible) {
+
+    //FIXME: As soon the backend exposes oc-share-root via webdav, only use isShareRoot fn
+    const shareRoot = isShareRoot(resource) || resource.id === space.id
```

Comment:
Arghs, `resource.id === space.id` is an issue for spaces. When you e.g. search for a space, then the space's parent is now "Shared with me". I guess we need this: `isShareRoot(resource) || (resource.id === space.id && resource.type === 'file')`

Reply:
@butonic is coming up with a fix, I think we can leave this open until we got it and remove the "gehampel 🤡 " along

Reply:
Ahh nice 👍

Reply:
Unfortantly the fix won't come, but I enhanced the condition